plugins/performance-lab/includes/admin/plugins.php

```diff
@@ -124,6 +116,61 @@ function perflab_render_plugins_ui(): void {
 		}
 	}
 
+	if ( count( $errors ) > 0 ) {
+		$active_plugins = array_map(
+			static function ( string $file ) {
+				return strtok( $file, '/' );
+			},
+			array_keys( get_plugins() )
+		);
+		$plugin_list    = '<ul>';
+		$error_messages = array();
+		foreach ( $errors as $plugin_slug => $error ) {
+			if ( defined( $standalone_plugin_data[ $plugin_slug ]['constant'] ) ) {
+				$status = __( '(active)', 'performance-lab' );
+			} elseif ( in_array( $plugin_slug, $active_plugins, true ) ) {
+				$status = __( '(installed)', 'performance-lab' );
+			} else {
+				$status = '';
+			}
+
+			$plugin_list     .= sprintf(
+				'<li><a target="_blank" href="%s"><code>%s</code></a> %s</li>',
+				esc_url( trailingslashit( __( 'https://wordpress.org/plugins/', 'default' ) . $plugin_slug ) ),
+				esc_html( $plugin_slug ),
+				esc_html( $status )
+			);
+			$error_messages[] = $error->get_error_message();
+		}
+		$plugin_list .= '</ul>';
+
+		$error_messages = array_unique( $error_messages );
+
+		wp_admin_notice(
+			esc_html(
```

Comment:
@mukeshpanchal27 do we need this now?

```suggestion
			'<p>' . esc_html(
```